pulp_container/app/models.py

```diff
@@ -355,13 +355,17 @@ class ContainerRemote(Remote, AutoAddObjPermsMixin):
         exclude_tags (fields.ArrayField): List of tags to exclude during sync.
         sigstore (models.TextField): The URL to a sigstore where signatures of container images
             should be synced from.
+        includes (models.JSONField): Dictionary of repositories to include. [default=null]
+        excludes (models.JSONField): Dictionary of repositories to exclude. [default=null]
     """
 
     upstream_name = models.TextField(db_index=True)
     include_foreign_layers = models.BooleanField(default=False)
     include_tags = fields.ArrayField(models.TextField(null=True), null=True)
     exclude_tags = fields.ArrayField(models.TextField(null=True), null=True)
     sigstore = models.TextField(null=True)
+    includes = models.JSONField(null=True)
+    excludes = models.JSONField(null=True)
```

Comment:
hum... I had left it there because the `get_or_create` from `get_pull_through_drv` was expecting it (if not provided, it was raising a `FieldError` exception):
https://github.com/pulp/pulp_container/blob/b126f4ca621e0559a00bf23e2d5e85c52ed65e44/pulp_container/app/registry_api.py#L318-L324

but, now that you mentioned it, I guess we can remove it and just delete the `includes`/`excludes` keys from `remote_data` dictionary before calling the `get_or_create`.

Reply:
Interesting. Are not these two models completely isolated?

Reply:
hum... kind of!?

from my understanding, when we create the `ContainerRemote` https://github.com/pulp/pulp_container/blob/b126f4ca621e0559a00bf23e2d5e85c52ed65e44/pulp_container/app/registry_api.py#L320-L325 we are passing the remote values (`remote_data`) extracted from the `ContainerPullThroughRemote` https://github.com/pulp/pulp_container/blob/ba2e297b8414e5f52474c35dcc8a5cada24a204e/pulp_container/app/registry_api.py#L319
https://github.com/pulp/pulp_container/blob/ba2e297b8414e5f52474c35dcc8a5cada24a204e/pulp_container/app/registry_api.py#L393-L399

Reply:
Aha, I see. Then, we may want to ignore these fields. Thus, we can alter `IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES` to suit our needs.